Tests/test_file_iptc.py

```diff
@@ -6,18 +6,21 @@
 
 from PIL import Image, IptcImagePlugin
 
-from .helper import hopper
+from .helper import assert_image_equal, hopper
 
 TEST_FILE = "Tests/images/iptc.jpg"
 
 
 def test_open():
+    expected = Image.new("L", (1, 1), 0)
```

Comment:
```suggestion
    expected = Image.new("L", (1, 1))
```
This is just my preference.

Reply:
Sure.